packages/account/src/Components/demo-message/__test__/demo-message.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DemoMessage from '../demo-message';
+
+// eslint-disable-next-line react/display-name
+jest.mock('Components/icon-with-message', () => () => <div>DemoMessage</div>);
```

Comment:
```suggestion
jest.mock('Components/icon-with-message', () => function DemoMessage() {
            return <div>DemoMessage</div>
        });
```
@prince-deriv a regular function declaration can help here against display-name eslint error 🙂